### PR TITLE
[#31] Improve the UX flow for the Log In mode switch button at the bo…

### PR DESCRIPTION
**Pull request by kroucis**
[#31] Improve the UX flow for the Log In mode switch button at the bottom of the "Join Odysee" view by changing the tappable label to a full-fledged rounded button.

New Sign Up view:
Simulator Screen Shot - iPhone 8 - 2021-07-21 at 03 04 09

New Log In view:
Simulator Screen Shot - iPhone 8 - 2021-07-21 at 03 04 21

### CODE REVIEW

**Review by akinwale — CHANGES_REQUESTED** [review]
This looks good, thanks. Could you make the button at the bottom half the length of the main button? Once you push up your commit, I'll merge the change.

**Comment by kroucis** [comment]
I had to get....destructive to get IB to allow the 1/2 width button request. It *really* did not like a 0.5 multiplier on the button while it was in the StackView (and I couldn't just add the 1/2 width constraint to the StackView because the label was effected too). So I ended up removing the StackView altogether and now the Log In button is the anchor for the label at the bottom.

New Sign Up view with half-width button
Simulator Screen Shot - iPhone 8 - 2021-07-21 at 12 17 46

New Log In view with half-width button
Simulator Screen Shot - iPhone 8 - 2021-07-21 at 12 18 25

**Comment by kroucis** [comment]
Bumping this thread. The button size change has been implemented and is ready for another review.

**Review by akinwale — APPROVED** [review]
Looks good! Thanks.

**Pull request merged**
